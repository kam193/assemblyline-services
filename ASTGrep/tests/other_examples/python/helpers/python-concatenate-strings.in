x = 'I' + (
        'IllllllllIlllIIIIlllIIll'
    )

"a" + "b" + "c" + "d"

("a") + ("b") + ("c") + ("d")

("a") + "b"

"a" + (3)

"a" + (a)

"a" + ("b", a)
