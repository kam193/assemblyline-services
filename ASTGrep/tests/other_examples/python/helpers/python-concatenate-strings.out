x = 'IIllllllllIlllIIIIlllIIll'

'abcd'

'abcd'

'ab'

"a" + (3)

"a" + (a)

"a" + ("b", a)
